backend/app/domains/users/routers.py

```
@@ -0,0 +1,136 @@
+import logging
+from datetime import datetime, timedelta
+from typing import Optional, List
+
+from fastapi import APIRouter, HTTPException, Depends, status
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+from jose import JWTError
+from jose import jwt
+from sqlmodel import Session
+
+from app.core.config import Settings
+from app.db.session import get_db as get_session, get_db
```

Pylint (3.3.7)

[warning] Reimport 'get_db' (imported line 12)

(W0404)